gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/test/java/io/gravitee/rest/api/management/rest/resource/EnvironmentAnalyticsResourceTest.java

```diff
@@ -73,7 +73,7 @@ public void shouldGetEmptyHistoAnalyticsWhenNotAdminAndNoApp() {
             .thenReturn(Collections.emptySet());
 
         Response response = envTarget()
-            .queryParam("type", "date_histo")
```

Comment:
I'm not very fan of changing the case. It will be a breaking change for users.
And if it's only in the test, then we should try to fix the test configuration.

Reply:
As I've noted several times, I'm not a fan of lax APIs. If an enum is defined then the client has to use exactly this value everything else is a bad request.